tools/qontract_cli.py

```diff
@@ -492,6 +492,27 @@ def quay_mirrors(ctx):
     print_output(ctx.obj['output'], mirrors, columns)
 
 
+@get.command()
+@click.argument('aws_account')
+@click.argument('db_instance_name')
```

Comment:
identifier? just to make it consistent with app-interface?

Reply:
fixed in 82485ad :)